fastify.js

```diff
@@ -56,7 +57,22 @@ function build () {
       throw new Error(`${opts.method} method is not supported!`)
     }
 
-    opts[schema] = fastJsonStringify(opts.schema.out)
+    // Missing handler => schema is a function
+    if (!opts.handler && typeof opts.schema === 'function') {
+      opts.handler = opts.schema
+      opts.schema = {}
+    }
```

Comment:
Ok initially that code was inside the shorthand functions but then I put it there and not in the shorthand function to avoid code repeat.
I'll update this :)